cli/src/commands.rs

```diff
@@ -125,257 +38,8 @@ pub enum Commands {
 
 pub fn match_command(cli: &Cli) {
 	match &cli.command {
-		Commands::Balance { account } => balance(cli, account),
-		Commands::NewAccount => new_account(),
-		Commands::ListAccounts => list_accounts(),
-		Commands::PrintMetadata => print_metadata(cli),
-		Commands::PrintSgxMetadata => print_sgx_metadata(cli),
-		Commands::Faucet { accounts } => faucet(cli, accounts),
-		Commands::Transfer { from, to, amount } => transfer(cli, from, to, amount),
-		Commands::ListWorkers => list_workers(cli),
-		Commands::Listen { events, blocks } => listen(cli, events, blocks),
-		Commands::ShieldFunds { from, to, amount, shard } =>
-			shield_funds(cli, from, to, amount, shard),
-		Commands::Trusted(trusted) => trusted_commands::match_trusted_commands(cli, trusted),
+		Commands::Integritee(cmd) => cmd.run(cli),
+		Commands::Trusted(cmd) => cmd.run(cli),
```

Comment:
Hmm, I am unsure about the naming here, but I don't really have a better suggestion:

* IntegriteeCore maybe?

Ah, I don't know, maybe we should leave it like it is if you prefer yours.

Reply:
No, I agree. `Integritee` and `Trusted` doesn't really make sense. Everything is `Integritee`, is it not?

What about : `General` and `Encrypted`? Or `Shielded`?